lib/dotcom/trip_plan/map.ex

```diff
@@ -150,4 +197,13 @@ defmodule Dotcom.TripPlan.Map do
   def z_index(%{current: idx, start: idx}), do: 100
   def z_index(%{current: idx, end: idx}), do: 100
   def z_index(%{}), do: 0
+
+  defp tailwind_color("#003DA5"), do: "blue-500"
```

Comment:
I'm torn about this. Most of the other places where we translate things from GTFS to color, we do it based on the route, but here we're pulling the color out of the polyline directly.

If it turns out that we can get this to work, should we use the colors directly in otther contexts too?

Reply:
The color that's in the polyline came from the associated route, it's just parsed roundaboutly because legacy....